esx_service/cli/vmdkops_admin.py

```diff
@@ -172,83 +175,186 @@ def commands():
+                    'cmds': {
+                        'add': {
+                            'func': tenant_access_add,
+                            'help': 'Add a datastore access for a tenant',
+                            'args': {
+                                '--name': {
+                                    'help': 'The name of the tenant',
+                                    'required': True
+                                },
+                                '--datastore': {
+                                    'help': "Datastore which access is controlled",
+                                    'required': True
+                                },
+                                '--rights': {
+                                    'help': 'Datastore access Permissions granted',
+                                    'choices': ['create', 'delete', 'mount', 'all'],
+                                    'metavar': 'create,delete,mount'
+                                },
+                                '--volume-maxsize': {
+                                    'help': 'Maximum size of the volume that can be created',
+                                    'metavar': 'Num{MB,GB,TB} - e.g. 2TB'
+                                },
+                                '--volume-maxcount': {
+                                    'help': 
+                                    'Maximum number of volumes to create on the datastore for this tenant'
+                                },
+                                '--volume-totalsize': {
+                                    'help': 
+                                    'Maximum total size of all volume that can be created on the datastore for this tenant',
+                                    'metavar': 'Num{MB,GB,TB} - e.g. 2TB'
+                                }
+                            }
+                        },
+                        'set': {
+                            'func': tenant_access_set,
+                            'help': 'Modify datastore access for a tenant',
+                            'args': {
+                                '--name': {
+                                    'help': 'Tenant name',
+                                    'required': True
+                                },
+                                '--datastore': {
+                                    'help': "Datastore name",
+                                    'required': True
+                                },
+                                '--add-rights': {
+                                    'help': 'Datastore access Permissions granted',
+                                    'choices': ['create', 'delete', 'mount', 'all'],
+                                    'metavar': 'create,delete,mount'
+                                }, 
+                                '--rm-rights': {
+                                    'help': 'Datastore access Permissions granted',
```

Comment:
Also, rm-rights must allow a way to specify a list of datastores or "all" datastores in one go.